examples/EnvironmentCheck.ipynb

```diff
@@ -0,0 +1,272 @@
+{
```

Comment:
<p>To avoid the impression that all the dependencies fetched here are required for itkwidgets, can please clarify here:</p><p> </p><p>required dependencies -&gt; example notebook dependencies</p><p>other notebooks -&gt; example notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/itkwidgets/pull/614/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>